src/app/src/components/FacilityDetailSidebarClaimedInfo.jsx

```diff
@@ -156,6 +158,44 @@ export default function FacilityDetailsSidebarClaimedInfo({
                 value={facility.female_workers_percentage}
                 label="Percentage of female workers"
             />
+            <ClaimInfoSection
+                label="Affiliations"
+                value={
+                    facility.affiliations && facility.affiliations.length
+                        && (
+                            <ul>
+                                {
+                                    orderBy(facility.affiliations, identity)
+                                        .map(affiliation => (
+                                            <li key={affiliation}>
+                                                {affiliation}
+                                            </li>
+                                        ))
+                                }
+                            </ul>
+                        )
+
+                }
+            />
+            <ClaimInfoSection
+                label="Certifications/Standards/Regulations"
+                value={
+                    facility.certifications && facility.certifications.length
+                        && (
+                            <ul>
+                                {
+                                    orderBy(facility.certifications, identity)
```

Comment:
We are using `orderBy` both here and in `ClaimedFacilityDetailsReducer.js`. Consider removing the sort from one or the other.

Reply:
The orderBy here is to ensure that the Facility Details screen display is in alpha order; the reducer version is to ensure that the list sent to the API on profile updates is alphabetical. In general the API doesn't really bother to alphabetize things on its own, where the lists are effectively just sets -- but it will send an update email if the list has the same elements but in a different order, which is why I added the guard on the reducer.

I think I may leave the orderBy in both places in the client just for some display guarantees. If it becomes problematic we can move the alphabetization into the API.

Reply:
👍